### PR TITLE
Fixed the toaster position relative to the floating action button.

### LINKED ISSUE #2066: Fix the positioning of the toaster so that it does not overlap the floating action menu
Currently the toaster appears below the floating action menu and also in the same position as the floating action menu. The `z-index` issue will be fixed in the molecule but we need to handle the position in the toaster itself.

297656584-927ea8a5-5d8c-4bf1-ae05-8d22ae96ce12

@akkshaylawrence _a

### PR DESCRIPTION
**Pull request by akkshaylawrence**
- Fixes #2066

**Description**

**Checklist**

- [x] I have made corresponding changes to the documentation.
- [x] I have updated the types definition of modified exports.
- [x] I have verified the functionality in some of the neeto web-apps.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added proper `data-cy` and `data-testid` attributes.
- [x] I have added the necessary label (`patch`/`minor`/`major` - If package publish
      is required).

**Reviewers**
@praveen-murali-ind _a

### CODE REVIEW

File: src/styles/components/_toast.scss [hunk 1]
```diff
@@ -35,6 +35,7 @@ body {
   .Toastify__toast-container {
     z-index: var(--neeto-ui-toastr-z-index);
     padding: 0;
+    bottom: 60px !important;
```
Comment (praveen-murali-ind) [inline src/styles/components/_toast.scss:38]: @akkshaylawrence _a Let’s revert this change. If we override the Toast container bottom offset to 60px, it will also affect other Toast positions. We can provide a fix from neetoMolecules instead.

cc @ajmaln

**Comment by akkshaylawrence** [comment]
Closing this PR as per the discussion with @praveen-murali-ind
